packages/yad/yad.pacscript

```diff
@@ -32,7 +32,7 @@ prepare() {
 }
 
 build() {
-	./configure "$FLAGS"
+	./configure " $FLAGS"
```

Comment:
Umm... @echometerain , Can you explain why is this change necessary?

Reply:
It wouldn't build unless I do this

It likely resolved to `./configure--flag1 --flag2` instead of `./configure --flag1 --flag2`